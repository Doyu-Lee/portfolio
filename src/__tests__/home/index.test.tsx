import { render } from '@testing-library/react';
import Home from '@/app/page';


describe('Home', () => {
  it('snapshot 테스트입니다.', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});

