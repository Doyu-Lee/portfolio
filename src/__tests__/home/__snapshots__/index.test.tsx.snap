// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot 테스트입니다. 1`] = `
<div>
  <main
    class="main"
  >
    <h2>
      Home Page
    </h2>
  </main>
</div>
`;
